Droidcon-Boston/app/src/main/res/layout/tweet_item_layout.xml

```diff
@@ -27,6 +27,10 @@
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:layout_marginStart="8dp"
+        android:maxLines="1"
+        android:ellipsize="end"
+        app:autoSizeTextType="uniform"
+        app:autoSizeMinTextSize="10sp"
```

Comment:
it might be worth creating a style for this since we're using it in a few places